### PR TITLE
Display Cairo 1.0 tabs as the default, primary tag on the docs

### PR DESCRIPTION
**Pull request by JameStark**
### Description of the Changes

Please add a detailed description of the change, whether it's an enhancement or a bugfix.
If the PR is related to an open issue please link to it.

### PR Preview URL

After you push a commit to this PR, a preview is built and a URL to the root of the preview appears in the comment feed.

Paste here the specific URL(s) of the content that this PR addresses.

### Check List

- [ ] Changes have been done against dev branch, and PR does not conflict
- [ ] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/650)

### CODE REVIEW

File: components/Starknet/modules/architecture_and_concepts/pages/Contracts/contract-abi.adoc [hunk 1]
```diff
@@ -425,6 +272,165 @@ Cairo 1.0::
...
+      }
+    ],
+    "type": "function"
+  }
+]
+----
+
 ====
 
 [NOTE]
```
Comment (stoobie) [inline components/Starknet/modules/architecture_and_concepts/pages/Contracts/contract-abi.adoc:436]: Suggestion:

> The Cairo 0 compiler is no longer maintained. For more information on compiling Cairo 1 contracts
compilation, see link:https://github.com/starkware-libs/cairo#compiling-starknet-contracts[Compiling Starknet Contracts] in the Cairo Git repository.

File: components/Starknet/modules/architecture_and_concepts/pages/Contracts/contract-storage.adoc [hunk 2]
```diff
@@ -16,38 +16,44 @@ The basic function for writing to storage writes value to key is:
...
 [source,js]
 ----
-storage_write_syscall(address_domain, address, value)
+storage_write(key, value);
 ----
+
 ====
 // todo add description explaining what address_domain is
 
 Both are system calls that can be imported by adding the line:
```
Comment (stoobie) [inline components/Starknet/modules/architecture_and_concepts/pages/Contracts/contract-storage.adoc:37]: Why does it say "both"? Only `storage_write_syscall` is mentioned. Should `storage_read_syscall` also be mentioned?

**Review by stoobie — CHANGES_REQUESTED** [review]
The examples look fine (I didn't review the examples themselves, just that the Cairo 1 ones appear by default.)

You need to take down the old system calls page, as per our conversation. Also if there are any other obsolete files, those should come down as well.

**Review by JameStark — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/650)* status: 0 of 3 files reviewed, 2 unresolved discussions (waiting on @stoobie)

___
*[`components/Starknet/modules/architecture_and_concepts/pages/Contracts/contract-abi.adoc` line 436 at r1](https://reviewable.io/reviews/starknet-io/starknet-docs/650#gh-1260798592:-N_96mlxB5rN602l7v02:b-896fix) ([raw file](https://github.com/starknet-io/starknet-docs/blob/c4ec375ede1ff1223fe34048b8c4968e654dc35d/components/Starknet/modules/architecture_and_concepts/pages/Contracts/contract-abi.adoc#L436)):*
<details><summary><i>Previously, stoobie (Steve Goodman) wrote…</i></summary><blockquote>

Suggestion:

> The Cairo 0 compiler is no longer maintained. For more information on compiling Cairo 1 contracts
compilation, see link:https://github.com/starkware-libs/cairo#compiling-starknet-contracts[Compiling Starknet Contracts] in the Cairo Git repository.



</blockquote></details>

Done.

**Pull request merged**